tests/automation_framework/tests/worker_tests/test_lookup.py

```diff
@@ -134,3 +134,31 @@ def test_worker_lookup_jsonrpc_different_version(self):
                 is TestStep.SUCCESS.value)
 
         logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.worker
+    @pytest.mark.worker_lookup
+    @pytest.mark.test_worker_lookup_diff_unit_length
+    @pytest.mark.listener
+    @pytest.mark.sdk
+    def test_worker_lookup_diff_unit_length(self):
+        request_file = os.path.join(
+            constants.worker_input_file,
+            "worker_lookup_diff_unit_length.json")
+
+        err_cd = self.test_obj.setup_and_build_request_lookup(
```

Comment:
Should this test be passing? Can you print the response you get when worker_type 11111 is provided?

Reply:
Yes test is passing. Response in listener code flow :
[10:46:09 INFO    src.utilities.submit_request_utility] **********Received Request*********
{"jsonrpc": "2.0", "method": "WorkerLookUp", "id": 3, "params": {"workerType": 1111111}}

[10:46:09 INFO    src.utilities.submit_request_utility] **********Received Response*********
{'result': {'totalCount': 0, 'lookupTag': '', 'ids': []}, 'id': 3, 'jsonrpc': '2.0'} 

Notice that totalCount is 0, lookupTag and ids, both are empty for invalid workerType. In contrast, with  workerType 1, receiving response -

{"jsonrpc": "2.0", "method": "WorkerLookUp", "id": 3, "params": {"workerType": 1}}

[10:52:02 INFO    src.utilities.submit_request_utility] **********Received Response*********
{'result': {'totalCount': 1, 'lookupTag': 'ce99e54310adf6d170d6197ac3032f64bcbfa8d583911ba38a96ee60e7926c07', 'ids': ['ce99e54310adf6d170d6197ac3032f64bcbfa8d583911ba38a96ee60e7926c07']}, 'id': 3, 'jsonrpc': '2.0'}